packages/uploadthing/src/sdk/index.ts

```diff
@@ -296,4 +301,26 @@ export class UTApi {
       "An unknown error occured while getting usage info.",
     );
   }
+
+  /** Request a presigned url for a private file(s) */
+  async getSignedURL(
+    fileKey: string,
+    opts?: {
+      /**
+       * How long the URL will be valid for.
+       * - Must be positive and less than 7 days (604800 seconds).
+       * - You must accept overrides on the UploadThing dashboard for this option to be accepted.
+       * @default app default on UploadThing dashboard
+       */
+      expiresInSeconds?: number;
```

Comment:
Would be kind of nice if this could be specified in seconds, minutes, hours, *or* days, and we do the translation. Probably a nightmare to type though

Reply:
i gotchu https://github.com/pingdotgg/uploadthing/pull/525/commits/da7eb5dcac7bab868dcc6218cb8633531b366aad